src/Components/CreateImageWizardV2/CreateImageWizard.tsx

```diff
@@ -110,17 +112,17 @@ const CreateImageWizard = ({ isEdit }: CreateImageWizardProps) => {
   const { data, isSuccess, isFetching, isError } =
     useListFeaturesQuery(undefined);
 
-  const snapshottingEnabled = useMemo(
-    () =>
-      !(
-        !isError &&
-        !isFetching &&
-        isSuccess &&
-        data?.snapshots?.accessible === false &&
-        data?.snapshots?.enabled === false
-      ),
-    [data, isSuccess, isFetching, isError]
-  );
+  const snapshottingEnabled = useMemo(() => {
+    if (isProd() && !isBeta()) return false;
```

Comment:
Actually, in light of https://github.com/osbuild/image-builder-frontend/commit/1b35b3f67543c959c59209654d9ee7f96dc21236 I think we should enable this in preview for both prod and stage. I'll add a commit for that.

Reply:
I meant preview _only_ if it wasn't clear :thinking: